__tests__/formats/scssVariables.test.js

```diff
@@ -62,5 +63,18 @@ describe('formats', () => {
       });
     });
 
+    it('should optionally use !default', done => {
+      var themeableDictionary = _.cloneDeep(dictionary),
+        formattedScss = formatter(dictionary),
+        themeableScss = "";
+
+      expect(formattedScss).not.toMatch("!default");
+
+      themeableDictionary.allProperties[0].themeable = true;
+      themeableScss = formatter(themeableDictionary);
+
+      expect(themeableScss).toMatch("#EF5350 !default;");
+      return done();
```

Comment:
We don't need `return done();` here, that is used if the test has some asynchronous code which we do for testing scss/less validity.

Reply:
Good to know! I should have looked more closely at what that was for in the other test.

Reply:
This has been resolved.